### DISCUSSION TITLE
How long does it take for new contributions to master to be released?

### DISCUSSION THREAD

**Discussion by evgenimelan** [espressif/arduino-esp32#9598] [Q&A] [answered]
Hello everyone,

I contributed for the first time to arduino-esp32, a small pin change to a 3rd party board. `430f1f45ee1a70464df1d733d2a5ddba7a36df29`
The commit was approved and merged into `espressif:master` 3 weeks ago.

Is it possible to estimate how long it will take for the change to be part of the next release? I assumed it would be part of the last release which was 5 days ago.

I don't want to seem impatient, I just want to know the process since this is my first time.

Cheers
Evgeni

**Comment by me-no-dev** [comment]
It will be part of the 3.0.0 release. 2.0.x releases will not be done anymore (and were released from different branch than master)

**Reply by evgenimelan** [reply]
Hello, I just noticed, that it was not part of the release of 3.0.0 the pin number is still the same. Could you please provide an update on the status of this pull request? If there are any issues or additional work needed on my end to facilitate its inclusion, I would be more than happy to assist.

**Reply by me-no-dev** [reply, marked as answer]
you are mistaken. The change is available in 3.0.0 and confirmed on my computer

**Reply by evgenimelan** [reply]
You are right, the issue was on our side. Sorry for the inconvenience!
